resources/js/tryitout.js

```diff
@@ -51,6 +51,17 @@ function cancelTryOut(endpointId) {
     }
 }
 
+function recursiveAddSearchParams(url, previousAppend, previousKey, obj) {
+    if (typeof obj === 'object') {
+        Object.keys(obj).forEach(objKey => {
+            const nextParam = !previousKey ? previousAppend : `${previousAppend}[${previousKey}]`;
```

Comment:
This is pretty confusing. In what scenario would there not be a key (aka `previousKey`)?